docs/guides/reference/ard-expanded-processing-extent.md

```diff
@@ -89,33 +94,32 @@ dc.find_datasets(product="ga_ls8c_ard_3", limit=10, final_ancillaries="nonstanda
 
-The following map represents the density of USGS Level 1 L1TP scenes over the extended area. These scenes are used as input for ARD generation and represent a higher geometric accuracy as ground control points and a digital elevation model is incorporated into ortho-processing. Locations with L1TP level 1 inputs are scenes which contain islands or land features which can be used as control. Locations entirely over shallow Ocean which have been added for bathymetric purposes have low geometric accuracy.
+The following map represents the density of USGS Level 1 L1TP scenes over the extended area. These scenes are used as input for ARD generation and represent a higher geometric accuracy as ground control points and a digital elevation model is incorporated into ortho-processing. Locations with L1TP Level 1 inputs are scenes that contain islands or land features that can be used as a control. Locations entirely over shallow Ocean that have been added for bathymetric purposes have low geometric accuracy.
 
 :::{figure} /_files/ard-expanded-processing-extent/LS5_7_8_9_L1TP_Density_and_Coverage.png
 :alt: A map of Australia and surrounding areas annotated with rectangles representing density and coverage.
 
-LS 5/7/8/9 L1TP Density and Coverage
+Landsat 5, 7, 8, and 9 L1TP Density and Coverage
 :::
 
 ## BRDF quality issues 
 
-[BRDF](/guides/about/glossary/#brdf) is a solar illumination correction which adjusts reflectance intensity in differing amounts in differing directions. The light that comes from the Sun, reflects off the Earth and is received by the sensor is adjusted based on viewing/solar angle geometries of MODIS data.
+[BRDF](/guides/about/glossary/#brdf) is a solar illumination correction that adjusts reflectance intensity in differing amounts in differing directions. The light that comes from the Sun reflects off the Earth and is received by the sensor is adjusted based on the viewing or solar angle geometries of MODIS data.
 
-Sometimes over offshore locations we have found BRDF values to be unphysical. We have implemented a BRDF solution over the offshore locations, which masks out data from solar illumination correction where Fvol/Fiso or Fgeo/Fiso > 1. This ensures that we do not violate the theoretical definition of these parameters, meaning the values are realistic.
+Sometimes over offshore locations, we have found BRDF values to be unphysical. We have implemented a BRDF solution over the offshore locations, which masks out data from solar illumination correction where $\frac{Fvol}{Fisor} > 1$ or $\frac{Fgeo}{Fiso} > 1$. This ensures that we do not violate the theoretical definition of these parameters, meaning the values are realistic.

```

Comment:
@OHehir-Aaron Can we please explain what "unphysical" means here for the benefit of our users who won't be familiar with the term? Or altneratively, add it to the Knowledge Hub glossary.